### DISCUSSION TITLE
Is it possible to tranform input URIs for the json output?

### DISCUSSION THREAD

**Discussion by zachsa** [fsspec/kerchunk#309] [Q&A] [answered]
Hi - new to Kerchunk and fsspec!

I want to use kerchunk to create json files that are mapped 1:1 with NetCDF files. This seems straightforward. On the input side I want to use an internal URI (i.e. `http://192.168.111.01/some/path/to/range/served/file.nc`), but I want the output to include public URIs. 

I know that I can run a replace on the json, but is there a better way?

**Comment by martindurant** [comment, marked as answer]
A replace seems perfectly reasonable to me! 
The basic mapping looks like {virtual_path: [real_path, offset, size]} or {virtual_path: [real_path]), so you could do the same with a comprehension.
